File: Programming Fundamentals/Labs And Homeworks/BasicSyntaxExercise/PrintandSum.java
package BasicSyntaxExercise;

import java.util.Scanner;

public class PrintandSum {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int firstNumber = Integer.parseInt(scanner.nextLine());
        int secondNumber = Integer.parseInt(scanner.nextLine());
        int sum = 0;
        for (int i = firstNumber; i <= secondNumber; i++) {
            System.out.print(i + " ");
            sum += i;

        }
        System.out.printf("%nSum: %d", sum);
    }
}

File: JS Front-End/JS Syntax Fundamentals - Exercise/printAndSum.js
function printAndSum(start, end) {
    let sum = 0;
    let output = '';
    for (let i = start; i <= end; i++) {
        output += i + ' ';
        sum += i;
    }
    console.log(output);
    console.log(`Sum: ${sum}`);
}

printAndSum(50, 60);